alerts/config.go

```diff
@@ -0,0 +1,158 @@
+package alerts
+
+// Data models for care team alerts.
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/tidepool-org/platform/data/blood/glucose"
+	"github.com/tidepool-org/platform/structure"
+)
+
+// Config models a user's desired alerts.
+type Config struct {
+	OwnerID         string         `json:"ownerID" bson:"ownerID"`
```

Comment:
Would it be better to use `userId` attribute for the owner? This would allow to standardize on a single field when trying to find records owned by a given user.

Reply:
Could go either way. My intention was to make it clearer what the roles of each of the two people were. They're both users after all.

Reply:
I'm going to stick with `ID` in go, but I'll use `Id` in the json/bson. Perhaps that what you meant anyway. After giving it some thought, this seems like the place to make that shift.